jaq-interpret/src/into_iter.rs

```diff
@@ -12,25 +12,22 @@ impl<I: Iterator, F: FnOnce() -> I> IntoIterator for Delay<F> {
 }
 
 #[derive(Clone)]
-pub enum Either<L, R> {
-    L(L),
-    R(R),
-}
+pub struct Either<L, R>(pub either::Either<L, R>);
```

Comment:
I prefer to keep my own `Either` type, because I hardly make use of anything that `either::Either` provides.